Makefile

```
@@ -0,0 +1,47 @@
+
+MAKEFILE_ROOT=$(shell pwd)
+help: ## Display this help menu
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+# install: Ensure that the Argo CD namespace exists, that Argo CD is installed, and that CRDs we are using are applied to the current cluster
+install-argo: ## Ensure that the Argo CD namespace exists, and that CRDs we are using are applied to the current cluster
+	kubectl apply -f $(MAKEFILE_ROOT)/backend/config/crd/bases
+	kubectl apply -f $(MAKEFILE_ROOT)/backend-shared/config/crd/bases/managed-gitops.redhat.com_operations.yaml
+	kubectl create ns argocd || true
+	kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-cd/release-2.2/manifests/crds/application-crd.yaml
+
+# start: start all the components
+# ensure goreman is installed, with 'go install github.com/mattn/goreman@latest'
+start:
+	~/go/bin/goreman start
+
+build: build-backend build-cluster-agent ## Build all the components - note: you do not need to do this before running start
+
+build-backend: ## Build backend only
+	cd $(MAKEFILE_ROOT)/backend && make build
+
+build-cluster-agent: ## Build cluster-agent only
+	cd $(MAKEFILE_ROOT)/cluster-agent && make build
+test: test-backend test-backend-shared test-cluster-agent ## Run tests for all components
```

Add a space to be aligned with the rest of the file.

```suggestion

test: test-backend test-backend-shared test-cluster-agent ## Run tests for all components
```